### ISSUE TITLE
"Not found" error when resource is deleted from NR platform

### ISSUE THREAD

**Issue by gregc-88** [newrelic/terraform-provider-newrelic#1881]
I'm seeing this same behavior on the current version of the provider (newrelic/newrelic v2.48.0 and v.2.48.1). 

- stdout is just "ERROR" 
- pulling debug logs shows the following errors:  `[ERROR] eval: *terraform.EvalRefresh, err: Not Found`  &`[ERROR] eval: *terraform.EvalSequence, err: Not Found`
- Removing the object from the state file fixes the issue, however there isn't a clear indicator of what the discrepancy in the state file is. This means I have manually inspect the objects in the web UI against the code to see what's missing.

_Originally posted by @gregc-88 in https://github.com/newrelic/terraform-provider-newrelic/issues/1430#issuecomment-1181187330_

**Comment by gregc-88** [comment]
Spun this into a new issue since the referenced issue was closed in Feb.

This is the second time I've found TF managed objects deleted by other staff with NR access. Aside from stern messaging stating not to do this, is there's a preferred best practice for easily finding missing objects? Or is that workflow not necessary when the provider is able to recognize objects are missing and need to be recreated?
